portfolio/src/main/java/com/google/sps/servlets/DeleteServlet.java

```diff
@@ -15,9 +15,10 @@
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    DatastoreService datastore = DataServlet.datastore;
-    List<Key> keys = DataServlet.keys;
-    datastore.delete(keys);
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    Query query = new Query("Comment").setKeysOnly();
```

Comment:
Are you missing an import here for `Query`?

Reply:
Oops I was.

Reply:
Note that there are more missing imports here. (DatastoreServiceFactory, for instance).

Try invoking `mvn compile` to ensure that your PR is building as expected.